src/AuthModule/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,7 +1,171 @@
+              <div className="form-outline position-relative text-start d-flex flex-wrap">
+                <label className="w-100 form-label fw-medium mb-0">
+                New Password
+                </label>
+                <input
+                  {...register("newPassword", passwordValidation)}
+                  type={!showPassword ? "text" : "password"}
+                  className="form-input form-control bg-transparent border-0 rounded-bottom-0  border border-bottom text-white p-1 pb-0 flex-grow-1"
+                  placeholder="Enter your New Password"
+                  aria-label="readonly input example"
+                />
+                <button
+                  onClick={togglePasswordVisibility}
+                  type="button"
+                  className="input-group-text border-0  bg-transparent position-absolute mt-4 end-0 p-2"
+                >
+                  <i
+                    className={`far ${
+                      showPassword ? "fa-eye-slash" : "fa-eye"
+                    } eye`}
+                  ></i>
+                </button>
+              </div>
+              {errors?.newPassword && (
+                <p className="  text-danger ">{errors?.newPassword?.message}</p>
+              )}
+            </div>
+
+            <div className=" one-input-group">
+              <div className="form-outline position-relative text-start d-flex flex-wrap">
+                <label className="w-100 form-label fw-medium mb-0">
+                Confirm New Password
+                </label>
+                <input
+                  {...register("confirmNewPassword", passwordValidation)}
+                  type={!showPassword ? "text" : "password"}
+                  className="form-input form-control bg-transparent border-0 rounded-bottom-0  border border-bottom text-white p-1 pb-0 flex-grow-1"
+                  placeholder="Confirm New Password"
+                  aria-label="readonly input example"
+                />
+                <button
+                  onClick={togglePasswordVisibility}
+                  type="button"
+                  className="input-group-text border-0  bg-transparent position-absolute mt-4 end-0 p-2"
+                >
+                  <i
+                    className={`far ${
+                      showPassword ? "fa-eye-slash" : "fa-eye"
+                    } eye`}
+                  ></i>
+                </button>
+              </div>
+              {errors?.confirmNewPassword && (
+                <p className="  text-danger ">{errors?.confirmNewPassword?.message}</p>
+              )}
+            </div>
+
+
+
+          </div>
+         
+
+          <div className=" text-center mt-5">
+            <button
+              type="submit"
+              className={`btn btn-warning text-center  text-white w-75 rounded-5  ${
+                isLoading ? "noClick" : ""
+              }`}
+            >
+              {isLoading ? <Loader /> : "Save"}
```

Comment:
Consider changing the text here to a "change password" or something more meaningfull.